<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>feaAnalysisPanel</class>
 <widget class="QWidget" name="feaAnalysisPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radio1DElement">
             <property name="text">
              <string>1D Element</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio2DElement">
             <property name="text">
              <string>2D Element</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio3DElement">
             <property name="text">
              <string>3D Element</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QComboBox" name="comboBox1DElement"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox2DElement"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox3DElement"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="loadMesh">
         <property name="text">
          <string>loadMesh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="buttonRun">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
